packages/web-app-files/src/components/AppBar/AppBar.vue

```diff
@@ -94,6 +94,23 @@
                     </oc-button>
                   </div>
                 </li>
+                <li
+                  v-for="(mimetype, key) in getMimeTypes"
+                  v-if="mimetype.allow_creation === true"
+                  :key="key"
+                >
+                  <div>
+                    <oc-button
+                      appearance="raw"
+                      justify-content="left"
+                      :class="['uk-width-1-1']"
+                      @click="showCreateResourceModal(false, mimetype.ext, false, true)"
+                    >
+                      <oc-icon :name="mimetype.icon || 'file'" />
+                      <span><span v-translate>'New '</span> + {{ mimetype.name }}</span>
```

Comment:
Could you please use `translate-params` and a template string instead? That way it'll have the full context in transifex for translations:
`<translate :translate-params="{name: userFullName}">Foo %{name}</translate>`